t/27-consoles-s3270.t

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/perl
+
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use Test::Mock::Time;
+use Test::MockModule 'strict';
+use Test::MockObject;
+use Test::Output qw(combined_like stderr_like);
+use Test::Warnings qw(:all :report_warnings);
+use POSIX qw(waitpid _exit);
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use consoles::console;
+use backend::baseclass;
+use consoles::VNC;
+use consoles::s3270;
+
+$bmwqemu::vars{ZVM_HOST} = "localhost";
+$bmwqemu::vars{ZVM_GUEST} = "guest";
+$bmwqemu::vars{ZVM_PASSWORD} = "password";
+
+my $ipc_run_mock = Test::MockModule->new('IPC::Run');
+my $vnc_mock = Test::MockModule->new('consoles::VNC');
+my $inet_mock = Test::MockModule->new('IO::Socket::INET');
+my $s = Test::MockObject->new->set_true(qw(sockopt fileno print connected close blocking));
+
+$vnc_mock->redefine(_read_socket => sub { substr(${$_[1]}, $_[3], $_[2]) = $s->mocked_read; length ${$_[1]} });
+$vnc_mock->redefine(login => 1);
+$inet_mock->redefine(new => $s);
+$ipc_run_mock->redefine(start => sub ($self, $in, $out, $err) {
+        $$out = "success\nconnet(localhost)\nok";
+        return bless({'KIDS' => [{'VAL' => $$in, 'PID' => '0', 'NUM' => 1, 'TYPE' => 'cmd', 'RESULT' => 1, 'OPS' => []}]}, 'IPC::Run');
+});
+$ipc_run_mock->redefine(pump => 1);
+
+my $backend_base = backend::baseclass->new();
+my $backend_mock = Test::MockModule->new('backend::baseclass')->redefine(request_screen_update => sub ($self, $args = undef) {
+        $self->{cmdpipe} = undef;    # ensure we'll exit the while loop after one iteration
```

Comment:
Seems this line isn't covered so ... why is it needed then?

Reply:
This was added for the disabled code that I commented but then removed in the latest commit and forgot to remove this mock. I will remove these two lines as well or I will add more tests needing these mocks